pkg/safepath/join_unsafe.go

```diff
@@ -0,0 +1,31 @@
+//go:build !linux && !windows
+// +build !linux,!windows
+
+package safepath
+
+import (
+	"github.com/sirupsen/logrus"
+	"path/filepath"
+)
+
+// This is unsafe and suspectible to Time-Of-Check Time-Of-Use attack.
+func Join(path, subpath string) (string, func() error, error) {
```

Comment:
I think I'd prefer to error out on systems that we don't have an implementation of a safe join.

Reply:
Done